src/main/java/com/basho/riak/client/core/query/crdt/types/RiakMap.java

```diff
@@ -27,37 +27,35 @@
 /**
  * Representation of the Riak map datatype.
  * <p>
- * This is an immutable map returned when querying Riak for 
+ * This is an immutable map returned when querying Riak for
  * a map datatype.
  * </p>
- * 
+ *
  * @author Dave Rusek <drusek at basho dot com>
  * @since 2.0
  */
 public class RiakMap extends RiakDatatype
 {
-
-    private final Map<BinaryValue, List<RiakDatatype>> entries =
-        new HashMap<BinaryValue, List<RiakDatatype>>();
+    private final Map<BinaryValue, List<RiakDatatype>> entries = new HashMap<>();
```

Comment:
I don't understand this change, why is it in with a PR that adds a NotFound field to the fetch response?